### PR TITLE
Rita 03

### PR DESCRIPTION
**Pull request by ritamwaikinda**
Manual merge

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -2,8 +2,13 @@ import React, { useState, useEffect } from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
 import NavigationBar from './components/LandingPage/NavigationBar';
 import CharacterIndex from './components/CharacterIndex/CharacterIndex';
+<<<<<<< HEAD
```
Comment (ErnieAtLYD) [inline client/src/App.jsx:5]: Ooops! These markings mean that a merge conflict is about to be checked in. You'll want to use the code BETWEEN the `<<<<` or `>>>>>` and the `======`, and choose one section of code OR another.

**Review by ErnieAtLYD — CHANGES_REQUESTED** [review]
Hi Rita! Looks like some merge conflicted code is about to be checked in. You'll want to make the appropriate changes to remove that merge conflict and add some commits to this.

**Review by saritasalazar — APPROVED** [review]
Changes were made and updated. Looks good!

File: client/src/App.jsx (hunk 1 above)
Comment (ritamwaikinda) [reply client/src/App.jsx]: Changes have been made and updated

**Pull request merged**